core/arch/arm/kernel/link.mk

```diff
@@ -221,6 +221,26 @@ $(link-out-dir)/tee.bin: $(link-out-dir)/tee-pager.bin \
 		--tee_pageable_bin $(link-out-dir)/tee-pageable.bin \
 		--out $@
 
+all: $(link-out-dir)/tee-header-v2.bin
+cleanfiles += $(link-out-dir)/tee-header-v2.bin
+$(link-out-dir)/tee-header-v2.bin: $(link-out-dir)/tee.bin
```

Comment:
Hello, let me explain it; and please give comments :)
We make tee.bin as input and created another scripts due to:
1. init_size for 'tee_pager' in gen_hanshed_bin actually depends on files size of tee.bin in gen_hashed_bin.py; or we need to add more variables to calculate it size out while we generating other binaries (while generating these binaries tee.bin is not there so we need calculation);
2. Too much modification in .py and link.mk will make it looks quite a mess? We want to make the difference tiny :)

We will update another version which uses one .py file for final result and let's compare the difference?